package.json

```diff
@@ -12,8 +12,9 @@
     "node": ">=16.18.0"
   },
   "scripts": {
-    "start": "nodemon -e ts,tsx,json,js,jsx --watch ./src --ignore ./dist --exec 'yarn build'",
+    "start": "yarn typechain && nodemon -e ts,tsx,json,js,jsx --watch ./src --ignore ./dist --exec 'yarn dev'",
     "build": "yarn run clean && yarn typechain && yarn run build:cjs & yarn run build:esm & yarn run build:types; wait",
```

Comment:
i did not want to alter the current build/ci workflow by changing this script. we also still want to run typechain when we build and publish so instead created a new `dev` script that excludes the typechain command